components/kiwihr/actions/update-employee-record/update-employee-record.mjs

```diff
@@ -0,0 +1,197 @@
+        "employmentStartDate",
+      ],
+      optional: true,
+    },
+    aboutMe: {
+      propDefinition: [
+        kiwihr,
+        "aboutMe",
+      ],
+      optional: true,
+    },
+    gender: {
+      propDefinition: [
+        kiwihr,
+        "gender",
+      ],
+      optional: true,
+    },
+    managerId: {
+      propDefinition: [
+        kiwihr,
+        "managerId",
+      ],
+      optional: true,
+    },
+    nationality: {
+      propDefinition: [
+        kiwihr,
+        "nationality",
+      ],
+      optional: true,
+    },
+    teamIds: {
+      propDefinition: [
+        kiwihr,
+        "teamIds",
+      ],
+      optional: true,
+    },
+    positionId: {
+      propDefinition: [
+        kiwihr,
+        "positionId",
+      ],
+      optional: true,
+    },
+    locationId: {
+      propDefinition: [
+        kiwihr,
+        "locationId",
+      ],
+      optional: true,
+    },
+    birthDate: {
+      propDefinition: [
+        kiwihr,
+        "birthDate",
+      ],
+      optional: true,
+    },
+    personalPhone: {
+      propDefinition: [
+        kiwihr,
+        "personalPhone",
+      ],
+      optional: true,
+    },
+    personalEmail: {
+      propDefinition: [
+        kiwihr,
+        "personalEmail",
+      ],
+      optional: true,
+    },
+    addressStreet: {
+      propDefinition: [
+        kiwihr,
+        "addressStreet",
+      ],
+      optional: true,
+    },
+    addressCity: {
+      propDefinition: [
+        kiwihr,
+        "addressCity",
+      ],
+      optional: true,
+    },
+    addressState: {
+      propDefinition: [
+        kiwihr,
+        "addressState",
+      ],
+      optional: true,
+    },
+    addressPostalCode: {
+      propDefinition: [
+        kiwihr,
+        "addressPostalCode",
+      ],
+      optional: true,
+    },
+    addressCountry: {
+      propDefinition: [
+        kiwihr,
+        "addressCountry",
+      ],
+      optional: true,
+    },
+    pronouns: {
+      propDefinition: [
+        kiwihr,
+        "pronouns",
+      ],
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    try {
+      const {
+        kiwihr,
+        employeeId,
+        addressStreet,
+        addressCity,
+        addressState,
+        addressPostalCode,
+        addressCountry,
+        ...user
+      } = this;
+
+      const address = {};
+      if (addressStreet) address.street = addressStreet;
+      if (addressCity) address.city = addressCity;
+      if (addressState) address.state = addressState;
+      if (addressPostalCode) address.postalCode = addressPostalCode;
+      if (addressCountry) address.country = addressCountry;
+
+-     if (Object.keys(address) > 0) {
++     if (Object.keys(address).length > 0) {
+        user.address = address;
+      }
+
+      const response = await kiwihr.updateEmployee({
+        id: employeeId,
+        user,
+      });
+
+      $.export("$summary", `Successfully updated employee record for ID: ${this.employeeId}`);
```

Comment:
```suggestion
      $.export("$summary", `Successfully updated employee record for ID: ${employeeId}`);
```

Reply:
It's not necessary, since I already got it on line 164

Reply:
That was my point. `employeeId` is already declared on line 164, so you don't need `this` in front of it.

Reply:
I'll move to QA since it doesn't affect functionality.